### ISSUE TITLE
WebKit not support -webkit-image-rendering

### ISSUE THREAD

**Issue by yisibl** [postcss/autoprefixer#440]
input:

```css
.foo{
  image-rendering: crisp-edges;
}
```

Now output:
```css
.foo{
  -webkit-image-rendering: -webkit-optimize-contrast;
      -ms-interpolation-mode: nearest-neighbor;
          image-rendering: -moz-crisp-edges;
          image-rendering: -o-crisp-edges;
          image-rendering: crisp-edges;
}
```

> But WebKit not support -webkit-image-rendering

## We will output:

```css
.foo{
      -ms-interpolation-mode: nearest-neighbor;
          image-rendering: -webkit-optimize-contrast;
          image-rendering: -moz-crisp-edges;
          image-rendering: -o-crisp-edges;
          image-rendering: crisp-edges;
}
```

## See also: 

* http://trac.webkit.org/changeset/87121
* http://phrogz.net/tmp/canvas_image_zoom.html

**Comment by ai** [comment]
Done https://github.com/postcss/autoprefixer-core/commit/4d2c71afe021227a7dd977c7547e29eba0011caa

**Comment by ai** [comment]
Released in 5.1.10

**Comment by yisibl** [comment]
Good job!
